src/WebJobs.Script/Description/FunctionInvokerBase.cs

```diff
@@ -28,15 +28,14 @@ public abstract class FunctionInvokerBase : IFunctionInvoker, IDisposable
         private IMetricsLogger _metrics;
         private IDisposable _fileChangeSubscription;
 
-        internal FunctionInvokerBase(ScriptHost host, FunctionMetadata functionMetadata, ITraceWriterFactory traceWriterFactory = null)
+        internal FunctionInvokerBase(ScriptHost host, FunctionMetadata functionMetadata)
         {
             Host = host;
             Metadata = functionMetadata;
             _metrics = host.ScriptConfig.HostConfig.GetService<IMetricsLogger>();
 
             // Function file logging is only done conditionally
-            traceWriterFactory = traceWriterFactory ?? new FunctionTraceWriterFactory(functionMetadata.Name, Host.ScriptConfig);
-            TraceWriter traceWriter = traceWriterFactory.Create();
+            TraceWriter traceWriter = host.FunctionTraceWriterFactory.Create(functionMetadata.Name);
```

Comment:
I was actually trying to make it more consistent -- see directly above for how we pull the `IMetricsLogger`, for example, and then below how we pull `TraceWriter` and `LoggerFactory` from the host. I wasn't sure why `ITraceWriterFactory` was passed in via constructor while the rest were pulled from the host itself. I believe it was only used by one test as well, which made me think it wasn't a big deal to change that.